packer/install-dependencies.sh

```diff
@@ -363,6 +363,19 @@ for KUBERNETES_VERSION in ${K8S_VERSIONS}; do
     echo "  - ${CONTAINER_IMAGE}" >> ${RELEASE_NOTES_FILEPATH}
 done
 
+# TODO: fetch supported k8s versions from an aks-engine command instead of hardcoding them here
```

Comment:
you can remove that TODO line, it doesn't apply to Azure Stack versions since it's a subset of versions.

Reply:
I am hoping that Azure Stack versions can be validated some day in a similar way.